article/app/views/package.scala

```diff
@@ -36,7 +38,7 @@ object MainCleaner {
 object BodyCleaner {
   def apply(article: Article, html: String, amp: Boolean)(implicit request: RequestHeader) = {
     implicit val edition = Edition(request)
-    val cleaners = List(
+    val cleaners = ListBuffer(
```

Comment:
The problem with using an immutable list in this case is you'll have to make a bunch of copies and temporary vals, or use weird filtering. Unless I am missing something...